https://issues.redhat.com/browse/ACM-21283 (#7965)
open-cluster-management/rhacm-docs: global_hub/global_hub_requirements.adoc

|{acm-short} on IBM Power Systems | Yes | Yes
|===

* {global-hub} supports middleware, including: Kafka, PostgreSQL, and Grafana. The following table lists these middlewares and their built-in versions:

+
|===
|Middleware | Built-in version 

> 

Reviewer: Please add a new line for amq-streams-2.9.x
`| amq-streams | 2.9.x `

|Kafka | 3.9.0
|PostgreSQL | 16

> |Grafana | 2.12.0

Reviewer: Update the Grafana version to `11.1.0 `

|{acm-short} on IBM Power Systems | Yes | Yes
|===

> * {global-hub} supports middleware, including: Kafka, PostgreSQL, and Grafana. The following table lists these middlewares and their built-in versions:

Reviewer 2: Please try not to use possessive, this is also in the style guidance.

> * {global-hub} supports middleware, including: Kafka, PostgreSQL, and Grafana. The following table lists these middlewares and their built-in versions:

Reviewer 2: ```suggestion
* {global-hub} supports middleware, including: Kafka, PostgreSQL, and Grafana. The following table lists supported middleware and built-in versions:
```
